src/syntax/instrumentPpx.ml

```diff
@@ -65,9 +90,13 @@ let marker file ofs kind marked =
    unmodified if the expression is already marked, has a ghost location,
    construct instrumentation is disabled, or a special comments indicates to
    ignore line. *)
```

Comment:
Sure, one commit to clean these up is fine. I was going to do that like last time but I'm a bit pressed for time at the moment.